unison-src/transcripts/fix2254.output.md

```diff
@@ -86,30 +86,33 @@ scratch/a2> view A NeedsA f f2 f3 g
     = NeedsA (A a b Nat Nat)
     | Zoink Text
   
-  f : A Nat Nat Nat Nat -> Nat
+  f : #re3rf9cedk Nat Nat Nat Nat -> Nat
```

Comment:
This needs looking into 🤔

Reply:
I investigated this, it turns out that this type isn't being properly propagated to, and that this error exists on trunk as well, but appears to be masked by global names somehow.